tests/integration/component.go

```diff
@@ -816,18 +816,6 @@ func componentTests(args ...string) {
 	Context("odo component delete should clean owned resources", func() {
 		appName := "app"
 		cmpName := "nodejs"
-		var oc helper.OcRunner
-
-		JustBeforeEach(func() {
-			project = helper.CreateRandProject()
```

Comment:
The ```project``` is being used locally to the spec and moreover there is no such ```project``` is created in ```BeforeEach(func()```. So you can not remove it.